Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,16 +1,19 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.IO;
-using System.Linq;
-using System.Text;
+﻿using System.IO;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
+            var file = "./Input/b.txt";
+         //   var foo = File.ReadAllText(file);   
+
+
+          //  System.Console.WriteLine(FileCleaner.IsFileClean(foo));
+            FileCleaner.Clean(file, "./Input/ttft.txt");
+
+         //   System.Console.WriteLine(foo);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)